.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!--
+  Thank you for contributing to our project!
+  Your PR will be reviewed shortly after submitted.
+
+  Make sure you've read the contributing guidelines and our code of conduct.
+-->
+
+### Description
+
+<!-- Provide a description of the introduced changes in this pull request. -->
+<!-- If the pull request closes an issue, please update the issue ID below. -->
+
+Closes #{ISSUE_ID} <!-- Remove if not applicable -->
+
+### Checklist
+
+- [ ] A description of the changes in this PR is mentioned above.
+- [ ] All the new and existing test pass.
+- [ ] No plagiarized, duplicated, or repetitive documentation that has been directly copied from another source.
+- [ ] If it's a new explanation, it contains solid, understandable, and accessible information.
+- [ ] I have read the whole [contributing guidelines](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/CONTRIBUTING.md) and agree to the [Code of Conduct](https://github.com/TheAlgorithms/.github/blob/master/CODE_OF_CONDUCT.md).
+
+### Screenshots (if any)
```

Comment:
I'd probably remove this section - I haven't yet seen an explanation PR that require(d/s) screenshots. If someone really needs it, they can add it themselves.

Reply:
But what if the PR is used for something else, such as a new feature?
This PR template is not necessarily only for new explanations.

Reply:
Alright